go/vt/mysqlctl/backup.go

```diff
@@ -85,8 +89,7 @@ var (
 	// once before the writer blocks
 	backupCompressBlocks = 2
 
-	backupDuration  = stats.NewGauge("backup_duration_seconds", "How long it took to complete the last backup operation (in seconds)")
-	restoreDuration = stats.NewGauge("restore_duration_seconds", "How long it took to complete the last restore operation (in seconds)")
+	titleCase = cases.Title(language.English).String
```

Comment:
we need to document these in release notes about the deprecation and new counters..

Reply:
Have added to release notes